src/main/java/org/mitre/synthea/helpers/Utilities.java

```diff
@@ -115,6 +115,15 @@ public static int getMonth(long time) {
     return calendar.get(Calendar.MONTH) + 1;
   }
 
+  /**
+   * Get the the number of days from a Unix timestamp.
+   */
+  public static double getDurationDays(long time) {
```

Comment:
This method does not behave the way that I would expect. Given that it returns a `double`, I expect that it would return fractions of days. When passing in the value `43200000`, it returns `0.0`, not `0.5` as I would expect.

I recommend either changing the return type to `long` if this is the intended behavior. If not, dividing two longs will provide a whole number. Casting it to a `double` after this division will not provide additional precision.

Reply:
resolved by using BigDecimals

Reply:
Is there code that still needs to be checked in? I still see it using `long` values.

Reply:
Sorry.. somehow was looking at an older commit.